src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_5_layout.html.twig

```diff
@@ -6,7 +6,11 @@
     {%- set prepend = not (money_pattern starts with '{{') -%}
     {%- set append = not (money_pattern ends with '}}') -%}
     {%- if prepend or append -%}
-        <div class="input-group{{ group_class|default('') }}">
+        {%- set inputClass = ['input-group'] -%}
+        {% if group_class is defined %}
+            {% set inputClass = group_class|merge(inputClass) %}
+        {% endif %}
+        <div class="{{ inputClass | join(' ')}}">
```

Comment:
What was wrong with `<div class="input-group {{ group_class|default('') }}">`?

Reply:
The above renders as <div class="input-group "> which fails the tests. 
Test requires an exact match with <div class="input-group"> (note the space in the class.)

Reply:
is this a test shared between different themes ? If no, I suggest amending the test to match the new behavior.

Reply:
I seem to be coming up against a deprecation issue around the fact that group_class is not defined, or is able to be undefined. So later versions of 8.2 are failing on legacy deprecation notices.
Trying to filter by length or 'is defined' raises a null type error.
Not having much experience with twig, I'm suspecting my solutions are looking hacky!

Reply:
> is this a test shared between different themes ? If no, I suggest amending the test to match the new behavior.

It seems it's an issue across multiple themes. But that they each have their own block.

Reply:
My question is not whether the issue is shared between themes (which is a separate question, even though it is an interesting one). My question is whether the tests rejecting your initial patch (adding a space all the time after `input-group`) is shared with other themes or if its specific to this theme. If it is specific to this theme, we could patch the test as well and use the simple patch (which is the simpler code and so my preferred choice)

Reply:
Tests are specific to the respective theme, I'll revert back to the simpler version and modify the tests. Thanks for the steer on your prefence.

Reply:
If you switch back to the `<div class="input-group {{ group_class|default('') }}">`, there are two tests in `AbstractBootstrap5LayoutTest` and `FormExtensionBootstrap5LayoutTest` that would need to be adjusted. I think I agree with @stof that doing that change is good enough.